### ISSUE TITLE
Ability to disable Auto-cloaking

### ISSUE THREAD

**Issue by martin-braun** [dotenv-org/dotenv-vscode#25]
I miss the ability to disable the Auto-cloaking feature. Not everybody who is working with .env files is streaming or has to worry about visually leaked data.

**Comment by motdotla** [comment]
@martin-braun I agree. I started work on this. Will go out as `0.8.0` relatively shortly.

**Comment by motdotla** [comment]
Also, @martin-braun can I ask how you knew to update the latest version that contained auto-cloaking? 

Did VSCode prompt you or did you choose to update the installation manually?

**Comment by martin-braun** [comment]
@motdotla I manually installed your application, because I saw the Twitter Tweet about Auto-Cloaking. I was curious and I like the concept when doing screen shares and working in public places, although it falls short.

I want to mention that it might be useful to have a quick toggle in the bottom bar as well as a flag in the extension settings itself. 

Finally, I think the concept would be better in a dedicated extension that goes beyond .env files, i.e. for secrets in dart files, AndroidManifest.xml (Google Maps API key), my.cnf (SQL credentials) and so on, but this would exceed the scope, since this is the DotEnv extension.

Overall, it's a good start. Kudos!
